deploy/kubernetes/releases/csi-digitalocean-dev.yaml

```diff
@@ -435,7 +434,7 @@ spec:
       hostNetwork: true
       containers:
         - name: driver-registrar
-          image: quay.io/k8scsi/driver-registrar:v0.4.1
+          image: quay.io/k8scsi/driver-registrar:v1.0-canary
```

Comment:
Should we use the new [node-driver-registrar](https://github.com/kubernetes-csi/node-driver-registrar/releases/tag/v1.0.1) here?

Reply:
Agree that it seems like we should use `node-driver-registrar` since `driver-registrar` is deprecated.

Reply:
Yeap, as I said this was splitted and needs to be changed. However, I"m waiting for the documentation on how to implement and add both the `cluster-` and `node-` driver registrars. The relevant PR that adds the instructions will be added today or tomorrow to the CSI documentation repo.